docs/hub-deployment-guide/runbooks/phase2/index.md

```diff
@@ -0,0 +1,74 @@
+(hub-deployment-guide:runbooks:phase2)=
+# Phase 2: Cluster setup
+
+This assumes all engineers have access to this new account, and will be able to set up the cluster and support, without any new hubs being set up.
+
+## Definition of ready
+
+The following table lists the information that needs to be available to the engineer before this phase can start.
+
+- Region / Zone of the cluster
+- Name of cluster
+- Is GPU required?
+
+## Outputs
+
+At the end of Phase 1, there should be a new cluster setup, with fully configured and deployed support components.
+
+The file assets that should have been generated at the end of this phase and included in the PR should be:
+
+```bash
+➕ config/clusters/<new-cluster-name>
+  ├── cluster.yaml
+  ├── enc-deployer-credentials.secret.json
+  ├── enc-support.secret.values.yaml
+  ├── enc-grafana-token.secret.yaml
+  └── support.values.yaml
+```
+
+```bash
+➕ terraform
+  └── <new-cluster>.terraform
```

Comment:
```suggestion
  └── <new-cluster>.tfvars
```

Reply:
I also think this should be `terraform/<cloud-provider>/projects/` rather than just `terraform/`